scripts/sync-kube-mixin.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+# make a temporary dir to work in
+TMPDIR=$(mktemp -d -t 'tmp.XXXXXXXXXX')
```

Comment:
Missing quotes. Did you use `shellcheck`? ;)
There's a few other vars without quotes in the script.